lib/rspec/core/option_parser.rb

```diff
@@ -291,6 +291,8 @@ def initialize_project_and_exit
     def bisect_and_exit(argument)
       RSpec::Support.require_rspec_core "bisect/coordinator"
 
+      # This might be the best spot to mutate ENV['SPEC_OPTS']
+      # if we do that.
```

Comment:
I didn't find an obvious spot in the ConfigurationOptions nor Configuration where this would go and then be available to the bisect runner